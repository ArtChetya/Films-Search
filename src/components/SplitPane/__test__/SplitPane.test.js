import { shallow } from 'enzyme/build';
import React from 'react';
import { SplitPane } from '..';

describe('SplitPane component', () => {
    it('should be rendered correctly', () => {
        const component = shallow(
            <SplitPane left={<div>Test 1</div>} right={<div>Test 2</div>} />
        );
        expect(component).toMatchSnapshot();
    });
});
