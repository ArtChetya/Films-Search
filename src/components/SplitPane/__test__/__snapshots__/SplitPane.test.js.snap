// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SplitPane component should be rendered correctly 1`] = `
<AppBar
  bgcolor="#eeeeee"
  minheight="20px"
  opacity={1}
>
  <WithStyles(Grid)
    alignItems="center"
    container={true}
  >
    <WithStyles(Grid)
      item={true}
      xs={6}
    >
       
      <div>
        Test 1
      </div>
       
    </WithStyles(Grid)>
    <WithStyles(Grid)
      container={true}
      item={true}
      justify="flex-end"
      xs={6}
    >
      <WithStyles(Grid)
        item={true}
      >
        <div>
          Test 2
        </div>
      </WithStyles(Grid)>
    </WithStyles(Grid)>
  </WithStyles(Grid)>
</AppBar>
`;
